### DISCUSSION TITLE
How can i click outside a Dialog when is open ?

### DISCUSSION THREAD

**Discussion by theo-mesnil** [ariakit/ariakit#2669] [Questions] [answered]
Hi,

When i open a dialog i need to keep element outside clickabled. This works good with reakit, but i can't reproduce with Ariakit.

https://github.com/ariakit/ariakit/assets/50322149/d9d39d2e-e7d0-4ba2-a179-ba0a213a146b

It is a way to do this ?

Old component on reakit: https://github.com/WTTJ/welcome-ui/blob/master/packages/Drawer/src/index.tsx
New component on ariakit: https://github.com/WTTJ/welcome-ui/pull/2101/commits/3d64b9d1fb7d8d61629311ca38bfd1b89b3b1672

Thanks :)

**Comment by diegohaz** [comment, marked as answer]
The Ariakit behavior is more accessible and better conforms to the "modal" state. This means that elements must be either accessible to everyone, including mouse, keyboard, and screen reader users, or inaccessible to everyone. In the past, clickable elements were only accessible to mouse users, which excluded other user groups.

You have two options:

- Set [`modal`](https://ariakit.org/reference/dialog#modal) to `false` on the `Dialog` component, so all elements outside are accessible to everyone.

- Return specific elements from the [`getPersistentElements`](https://ariakit.org/reference/dialog#getpersistentelements) prop so they're accessible to everyone.

**Reply by theo-mesnil** [reply]
Thanks it's perfect :) It works!
